### DISCUSSION TITLE
OLAF - Wake regularization parameter and Wing regularization factor

### DISCUSSION THREAD

**Discussion by Laurens09** [OpenFAST/openfast#2002] [Q&A]
Dear all,

could you explain me what are the Wake regularization factor (`WakeRegFactor`) and Wing regularization factor (`WingRegFactor`), and their role inside the OLAF module?

They are not explained in the OLAF theory section (https://openfast.readthedocs.io/en/dev/source/user/aerodyn-olaf/OLAFTheory.html).

Thank you
Best regards

**Comment by robytian** [comment]
Hi Laurens09

In my opinion, the regularization parameter is mainly used to define the vortex core growth model, that is, the variation of the induced velocity solution formula considering the effect of vortex stretching, dissipation and turbulence. One of the parameters involved is the change of the radius of the vortex core. Before the change of the radius of the vortex core, we need to set an initial radius of the vortex core, and WakeRegFactor and WingRegFactor here play a role in defining the radius of the vortex core, that is, the percentage of blade span or chord length. This definition was proposed by E. Branlard, but its essence is to define the size of the radius of the vortex core.
In addition, wingfactor represents the radius of the vortex core of the bound vortex on the blade, while wakefactor represents the radius of the wake. If your wakeregmethod uses a constant, then this value is fixed and determined according to your setting. However, we usually choose to change with the age Angle, that is, the wakeregmethod chooses 3. In this way, we do not need to consider the wakefactor, but only the vortex core radius of the initial bound vortex on the blade given by wingfactor.
Hope these help you.

Robytian

**Reply by Laurens09** [reply]
Thank you for your answer. 

I have a question: how does size of the radius of the vortex core influence the model? What is the difference between a model with a long radius and a smal one?

**Reply by robytian** [reply]
Hi
Because the vortex core growth model is also an empirical formula, it is sufficient to take 10% of the chord length as recommended, which has little impact on the model results, as long as it is not too outrageous.
BR,
roby

**Comment by ebranlard** [comment]
`WakeRegParam` is the old name for `WakeRegFactor` in the documentation. It was renamed to "Factor" because the parameter can have different meaning. We have forgotten some instances of `WakeRegParam` in the documentation. 

The "*RegFactors" are used to determine the initial core radius "r_c" of the vortex filaments. The smaller the core radius, the more "singular" the vortex (strong influence to its close neighbor). You don't want a radius very large at the wing and near wake as this will be not be consistent with the original lifting-line theory as the trailed vortices won't have enough influence on their close-neighbors, but a lot of small influence on many neighbors. 

The core growth model increases the core radius with time, which approximates the effect of diffusion. It spreads the vorticity into a bigger volume, and makes the velocity near a vortex element smaller. The model also introduce some numerical regularization (not necessarily physical) by making the wake less distorted as elements get closer together in the wake. 

You can find some ressources here: 

https://openfast.readthedocs.io/en/dev/source/user/aerodyn-olaf/InputFiles.html#wake-regularization-and-diffusion-options

https://openfast.readthedocs.io/en/dev/source/user/aerodyn-olaf/OLAFTheory.html#regularization
